packages/ListBoxWithTags/src/components/TriggerWithTags/TriggerWithTags.js

```diff
@@ -88,6 +101,37 @@ const renderTrigger = ({
       </sc.TriggerLabel>
     </>
   );
+}
+
+TriggerWithTags.displayName = "ListBoxWithTags.TriggerWithTags";
+
+TriggerWithTags.propTypes = {
```

Comment:
IMHO, most of these props don't make sense to declare them, the wrapper can't control them are just spread. 

And can be easily out of sync, neither provide any useful information to the consumer and don't see how much value add to the author.

Just my opinion.